tests/unit/test_attribute_mapping.py

```diff
@@ -37,6 +45,17 @@ def test_resource(self):
         expected = self.mappings_json["unifiedAttributeName"]
         self.assertEqual(expected, test.unified_attribute_name)
 
+    @responses.activate
+    def test_delete(self):
+        general_url = (
+            "http://localhost:9100/api/versioned/v1/projects/4/attributeMappings"
+        )
+        specific_url = "http://localhost:9100/api/versioned/v1/projects/4/attributeMappings/19629-12"
+        responses.add(responses.DELETE, specific_url, json=self.delete_json)
```

Comment:
`DELETE` endpoints have empty response bodies, so you shouldn't pass a `json` parameter. Instead, use `status=204`